### ISSUE TITLE
🐛 `RuntimeError: Insufficient resources available for job`

### ISSUE THREAD

**Issue by shnizzedy** [FCP-INDI/C-PAC#1511]
**Describe the bug**

Trying to run a high-resolution dataset with multiple functional runs with a many-forks pipeline config threaded wants to allocate too many resources.

**To Reproduce**
* Run [prelim_forks_ses_retest.yml](https://github.com/FCP-INDI/cpac_run_logs/files/6305066/prelim_forks_ses_retest.yml.txt) on a subject with multiple runs with 6 cores and --mem_gb 90
* with 6 cores and --mem_gb 110
* with 4 cores and --mem_gb 110

**Versions** 
 - C-PAC: 1.8.1-dev@​https://github.com/FCP-INDI/C-PAC/commit/b0593f9f2b5856624b4abc16530d6a1766985770

**Comment by shnizzedy** [comment]
In https://github.com/FCP-INDI/C-PAC/commit/7b237a289d8ecd5201d01c5b2d51a5de6dd469d5 I customized the prerun check to specify which nodes are estimating too much resource usage. 

C-PAC was estimating 240 GB for this function https://github.com/FCP-INDI/C-PAC/blob/7b237a289d8ecd5201d01c5b2d51a5de6dd469d5/CPAC/func_preproc/func_preproc.py#L466-L521 which is obviously way too much. I restored that node's memory estimate to the default (2 GB) which should resolve this issue.
